integrations/mistral/pyproject.toml

```
@@ -23,7 +23,7 @@ classifiers = [
   "Programming Language :: Python :: Implementation :: CPython",
   "Programming Language :: Python :: Implementation :: PyPy",
 ]
-dependencies = ["haystack-ai>=2.15.1"]
+dependencies = ["haystack-ai>=2.18.0"]
```

This needs to be 
```suggestion
dependencies = ["haystack-ai>=2.19.0"]
```
if you are also updating to use the `ToolsType`